inlong-manager/manager-service/pom.xml

```diff
@@ -557,5 +557,13 @@
             <artifactId>flink-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.hudi</groupId>
+            <artifactId>hudi-flink1.13-bundle</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-table-common</artifactId>
+        </dependency>
```

Comment:
Why add it in here

Reply:
flink-table-common is used to parse the schema of the hive table, has been rewritten, and removed the reference